lib/ansiblelint/rules/MetaMainHasInfoRule.py

```diff
@@ -4,16 +4,49 @@
 from ansiblelint.rules import AnsibleLintRule
 
 
+META_STR_INFO = [
+    'author',
+    'description',
+]
+META_INFO = META_STR_INFO + [
+    'license',
+    'min_ansible_version',
+    'platforms',
+]
+
+
+def _platform_info_errors_itr(platforms):
+    if not isinstance(platforms, list):
+        yield 'Platforms should be a list of dictionaries'
+        return
+
+    for platform in platforms:
+        if not isinstance(platform, dict):
+            yield 'Platforms should be a list of dictionaries'
+        elif 'name' not in platform:
+            yield 'Platform should contain name'
+
+
+def _galaxy_info_errors_itr(galaxy_info,
+                            info_list=META_INFO,
+                            str_info_list=META_STR_INFO):
+    for info in info_list:
+        ginfo = galaxy_info.get(info, False)
+        if ginfo:
+            if info in str_info_list and not isinstance(ginfo, str):
+                yield '{} should be a string'.format(info)
```

Comment:
explicit is more readable
```suggestion
                yield '{info} should be a string'.format(info=info)
```

Reply:
Absolutely right. Thanks for your advice!

I pushed the fix for this.